deepmd/utils/argcheck.py

```diff
@@ -717,8 +718,9 @@ def normalize(data):
     lra = learning_rate_args()
     la = loss_args()
     ta = training_args()
+    nvnmda = nvnmd_args()
 
-    base = Argument("base", dict, [ma, lra, la, ta])
+    base = Argument("base", dict, [ma, lra, la, ta, nvnmda])
```

Comment:
Besides this method, you may also need to modify `gen_doc` and `gen_json`:
https://github.com/deepmodeling/deepmd-kit/blob/c2e6b45eab20a1d933615dc1677fe701a61c5408/deepmd/utils/argcheck.py#L679-L683
https://github.com/deepmodeling/deepmd-kit/blob/c2e6b45eab20a1d933615dc1677fe701a61c5408/deepmd/utils/argcheck.py#L693-L699